index.html

```diff
@@ -8,13 +9,116 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Jost:wght@100&family=Just+Me+Again+Down+Here&family=Lora:ital,wght@1,500&family=Montserrat:wght@100&family=Nunito:wght@300&display=swap"
+    rel="stylesheet">
 </head>
+
 <body>
-  <header>
+  <header class="heading">
+    <div class="container">
+      <div class="row">
+        <div class="nav-bar">
+          <div class="column-one-fourth-left">
+            <button class="x-button"><i class="fa-solid fa-circle-xmark"></i></button>
+          </div>
+          <div class="column-full">
+            <h1>&#10024; HARRY POTTER CHARACTER CARDS &#10024;</h1>
+          </div>
+          <div class="column-one-fourth">
+            <button class="red-favorite-page"><i class="fa-solid fa-heart"></i></button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="information-container hidden">
+      <div data-view="information-view" class="information-row">
+        <div class="row">
+          <div class="column-full">
+            <h2 class="hp-name">Name</h2>
+            <div class="new-button-row">
+              <button class="like-button"><i class="fa-regular fa-heart"></i></button>
+            </div>
+          </div>
+          <div class="row img-placeholder">
+            <div class="column-full">
+              <img src="images/depositphotos_137014128-stock-illustration-user-profile-icon.jpg" alt="placeholder" />
+            </div>
+          </div>
+          <div class="column-half information-details">
+            <p class="born">
+              <span class="info-header">BORN</span>
+              <span class="born-info"></span>
+            </p>
+            <p class="died">
+              <span class="info-header">DIED</span>
+              <span class="died-info"></span>
+            </p>
+            <p class="blood">
+              <span class="info-header">BLOOD STATUS</span>
+              <span class="blood-status-info"></span>
+            </p>
+            <p class="house">
+              <span class="info-header">HOUSE</span>
+              <span class="house-info"></span>
+            </p>
+            <p class="patronus">
+              <span class="info-header">PATRONUS</span>
+              <span class="patronus-info"></span>
+            </p>
+            <p class="wand">
+              <span class="info-header">WAND</span>
+              <span class="wand-info"></span>
+            </p>
+            <p class="hp-wiki">
+              <span class="info-header">HARRY POTTER WIKI</span>
+              <span class="hp-info"></span>
+            </p>
+          </div>
+        </div>
```

Comment:
Where does all this appear on the page? If this is related to your next feature, then this shouldn't be here. Only the code related to this feature